.shared/.post-create

```diff
@@ -1,10 +1,18 @@
-To begin development, start Temporal Server:
+To begin development, install Docker:
 
-  {cyan cd} ~/path/to/temporal/docker-compose/
+  {cyan brew cask install docker}
+  {cyan open} /Applications/Docker.app
+  {cyan brew install docker-compose}
```

Comment:
i have a dumb question. how expensive would it be to check if docker is already installed, and/or if docker daemon is already running? feel like i'd like to get rid of this msg if its unnecessary.

Reply:
Don't know but good idea! Added:

https://github.com/temporalio/sdk-typescript/issues/541